updater/fetchers/debian/debian.go

```diff
@@ -67,8 +68,11 @@ func (fetcher *DebianFetcher) FetchUpdate(datastore database.Datastore) (resp up
 	r, err := http.Get(url)
 	if err != nil {
 		log.Errorf("could not download Debian's update: %s", err)
+		io.Copy(ioutil.Discard, r.Body)
 		return resp, cerrors.ErrCouldNotDownload
 	}
+	defer r.Body.Close()
+	r.Close = true
```

Comment:
useless, it has to be set before the request is done. `http.NewRequest()` must be used, then `r.Close = true` and then `http.DefaultClient.Do(request)`. Plus it presents TCP connections re-use and I don't we really need that.